packages/core/src/components/forms/numericInput.tsx

```diff
@@ -47,6 +47,14 @@ export interface INumericInputProps extends IIntentProps, IProps {
     disabled?: boolean;
 
     /**
+     * If set to `true`, the input will display with larger styling.
+     * This is equivalent to setting `pt-large` via className on the
+     * parent `pt-control-group` and on the child `pt-input-group`.
```

Comment:
I was copying the format of [`AbstractButton`](https://github.com/palantir/blueprint/blob/master/packages/core/src/components/button/abstractButton.tsx#L33). We should change both if I added a preceding `.` here. Should I go ahead and do that, or leave as is?